lib/modules/datasource/docker/index.ts

```diff
@@ -934,6 +934,9 @@ export class DockerDatasource extends Datasource {
       latestTag
```

Comment:
@rarkins what is the reason for using the `latestTag` here? I'm wondering if it would be possible to get the labels for the image that is being updated by Renovate in this particular PR instead of `latest` i.e. would this point to the wrong image (higher version)?

Reply:
I think cover one topic at a time, this may be a separate discussion/PR